modules/Emails/include/ListView/ListViewDataEmails.php

```diff
@@ -44,15 +44,31 @@
 
 require_once('include/ListView/ListViewData.php');
 
+include_once 'include/Exceptions/SuiteException.php';
+include_once 'modules/Emails/Folder.php';
+include_once 'modules/Emails/include/ListView/ListViewDataEmailsSearchOnCrm.php';
+include_once 'modules/Emails/include/ListView/ListViewDataEmailsSearchOnIMap.php';
+
+
+
 class ListViewDataEmails extends ListViewData
 {
+
+
```

Comment:
php doc added
(class ListViewDataEmailsSearchOnIMap and class ListViewDataEmailsSearchOnCrm not completely refactored yet)